admin/sql/messaging/CreateTables.sql

```diff
@@ -0,0 +1,42 @@
+\set ON_ERROR_STOP 1
+
+BEGIN;
+
+SET search_path = 'messaging';
+
+CREATE TABLE edit_note_thanks (
+  thanker             INTEGER NOT NULL, -- PK, references editor.id
+  edit_note		        INTEGER NOT NULL, -- PK, references edit_note.id
```

Comment:
This is mostly a stylistic suggestion, but usually when we have a compound table like `foo_bar` then `foo` is the first column and `bar` is the second. Same for other tables below.
```suggestion
CREATE TABLE edit_note_thanks (
  edit_note		        INTEGER NOT NULL, -- PK, references edit_note.id
  thanker             INTEGER NOT NULL, -- PK, references editor.id
```